src/pages/student_fees_list/FeesList.tsx

```diff
@@ -0,0 +1,277 @@
+        }
+      : skipToken
+  );
+
+  const handleOpenPayModal = (feeId: string) => {
+    setSelectedFeeId(feeId);
+    setModalType("pay");
+  };
+  const handleOpenPaymentInfoModal = (paymentId?: string) => {
+    setSelectedPaymentId(paymentId!);
+    setModalType("details");
+  };
+  const handleConfirmPay = async () => {
+    if (!selectedFeeId) return;
+    payMonthlyFee({
+      feeId: selectedFeeId,
+      studentId: student.id,
+    })
+      .unwrap()
+      .then(() => {
+        toast.success("Pago realizado correctamente");
+      })
+      .catch(() => {
+        toast.error("Ocurrió un error al procesar el pago");
+      })
+      .finally(() => {
+        setModalType(null);
+        setSelectedFeeId(null);
+      });
+  };
+
+  const handleCancelPay = () => {
+    setModalType(null);
+  };
+  if (isLoading) return <p>Cargando cuotas...</p>;
+  if (isError) return <p>Error al cargar cuotas.</p>;
+
+  const columns: Column<StudentMonthlyFeeResponse>[] = [
+    {
+      header: <SortableButton text="N° de cuota" />,
+      accessor: "number",
+    },
+
+    {
+      header: <SortableButton text="Mes" />,
+      accessor: "month",
+      Cell: ({ original }) => <span>{translateMonth(original.month)} </span>,
+    },
+
+    {
+      header: <SortableButton text={"Fecha de\nvencimiento"} allowWrap />,
+      accessor: "expirationDate",
+      Cell: ({ original }) => <span>{original.expirationDate}</span>,
+    },
+    {
+      header: <SortableButton text="Monto" />,
+      accessor: "amount",
+      Cell: ({ original }) => <span>{formatCurrency(original.amount)}</span>,
+    },
+    {
+      header: "Estado",
+      accessor: "status",
+      Cell: ({ original }) => (
+        <FeeStatusContainer>
+          <FeeStatus $status={original.status}>{original.status}</FeeStatus>
+        </FeeStatusContainer>
+      ),
+    },
+    {
+      header: "Pago",
+      Cell: ({ original }) => {
+        const canPay = ["Pendiente", "Vencida"].includes(original.status);
+        const canViewPayment = ["Pagado", "Pagado vencido"].includes(
+          original.status
+        );
+        if (canPay) {
+          return (
+            <ActionButton onClick={() => handleOpenPayModal(original.id)}>
+              Pagar
+              <CoinIconStyles src={CoinIcon} alt="coin-icon" />
+            </ActionButton>
+          );
+        }
+
+        if (canViewPayment) {
+          return (
+            <ActionButton
+              onClick={() => {
+                handleOpenPaymentInfoModal(original.paymentId);
+              }}
+            >
+              Ver pago
+              <ViewIconStyle src={ViewIcon} alt="view-icon" />
+            </ActionButton>
+          );
+        }
+        return <></>;
+      },
+    },
+  ];
+  return (
+    <StudentsContainer>
+      <InformationStudent>
+        <TitleContainer>
+          <img
+            src={BackIcon}
+            alt="back-icon"
+            onClick={() => navigate(-1)}
```

Comment:
Ojo con el navigate(-1), porque esto lo que hace es llevarte a la pagina previa en la que estuviste segun la pila de paginas que navegaste, si queremos ese comportamiento genial, ahora, si lo que queremos es que este boton te lleve a "Mis alumnos" para poder acceder a los datos de otro alumno, o lo que sea, esto estaria mal porque podra no llevarte a la seccion que deseamos, en ese caso tendriamos que poner la URL en vez de -1


```suggestion
            onClick={() => navigate(MisAlumnos)}
```